src/devices/Ads1115/Ads1115.cs

```diff
@@ -168,10 +168,10 @@ public double RawToVoltage(short val)
         /// </summary>
         public void Dispose()
         {
-            if (_sensor != null)
+            if (_i2cDevice != null)
             {
-                _sensor.Dispose();
-                _sensor = null;
+                _i2cDevice.Dispose();
```

Comment:
since you are already touching this, can we move this as we have it in the rest of the repo:

```csharp
_i2cDevice??.Dispose();
_i2cDevice = null;
```

Reply:
Sure.  Gimme a minute.  I found a couple others.